libraries/AP_Baro/LogStructure.h

```diff
@@ -32,6 +33,7 @@ struct PACKED log_BARO {
     float   drift_offset;
     float   ground_temp;
     uint8_t healthy;
+    float   corrected_pressure;
```

Comment:
Should log NaN if the feature is not avalalbe.

Reply:
Or I guess you could define it out of the structure completely.

Reply:
Fixed